test/tape/harness.js

```diff
@@ -35,6 +35,7 @@ module.exports = function(tableName, knex) {
         var val = cb(t)
         if (val && typeof val.then === 'function') {
           return val.catch(function(err) {
+            console.error(err);
```

Comment:
Was this intentional related to logging or can we remove it?

Reply:
Ah that's my bad. Stacktraces aren't printed when running tape and I had an issue in maria or oracle which I couldn't run locally at the time of making this, so abused Travis. 😄  I'll revert this back.